did_peer/src/peer_did/numalgo.rs

```diff
@@ -0,0 +1,36 @@
+use std::fmt::Display;
+
+use crate::error::DidPeerError;
+
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
This can be `Copy` and returning it would be cheap instead of passing around references.

Reply:
Can you please clarify for me - is the argument that cloning would be _cheaper_ than passing a reference?